azure-pipelines.yml

```diff
@@ -5,43 +5,43 @@
 
 #trigger only on master or PR to master
 trigger:
-- master
+  - master
```

Comment:
we build on circleci, azure and jenkins. They all use same gradle stuff so are equivalent. I intend to discontinue jenkins because of maintenance time and instability. I want to always have 2 because build is essential for production and I want to have fallback in case our primary build server would be discontinued. Because moving to another build server under pressure is underiable. And we can then also pinpoint build environment speicific issues.